src/main/java/mdlaf/components/password/MaterialPasswordFieldUI.java

```diff
@@ -23,6 +23,7 @@
  */
 package mdlaf.components.password;
 
+import mdlaf.components.textfield.MaterialTextFieldUI;
```

Comment:
This class needed more refactoring, I will create the new super Class for Put the code together